examples/map/brightness_pixel_location.py

```diff
@@ -23,7 +23,7 @@
 # then transform that pixel coordinate to a map coordinate.
 
 pixel_pos = np.argwhere(aia.data == aia.data.max()) * u.pixel
-hpc_max = aia.pixel_to_world(pixel_pos[:, 1], pixel_pos[:, 0])
+hpc_max = aia.wcs.pixel_to_world(pixel_pos[:, 1], pixel_pos[:, 0])
```

Comment:
This change should break this file, you will need to run it with `python examples/map/brightness_pixel_location.py` and fix any errors.

Reply:
That is good, I thought it would break but I was clearly wrong.

Reply:
> apparently these tests arent related to wcs , so i think we can go for the merge ?. updating examples and documentation can be handled in a separate issue

Go for the merge for what pull request?